.github/workflows/test.yml

```diff
@@ -72,3 +72,5 @@ jobs:
           name: dumps
           path: ${{ env.VSCODE_CRASH_DIR }}
         if: failure()
+      - name: Forbid TODOs
+        run: ./scripts/forbid-todo.sh
```

Comment:
I added to test workflow rather than pre-commit because if someone has our pre-commit hooks installed locally, we don't want to forbid them from commiting a TODO; we just want to ensure it doesn't get merged

Reply:
You could still commit skipping the pre-commit hook? i.e. `git commit --no-verify`

Reply:
We could, but I feel that fundamentally it's not a pre-commit requirement. It's a pre-merge requirement. Committing unfinished stuff is totally fine 🤷‍♂️